### DISCUSSION TITLE
Yaesu FT-710 CAT Control

### DISCUSSION THREAD

**Discussion by hb9hup** [magicbug/Cloudlog#2647] [Q&A]
Hello everyone.
I am new to this forum. I have successfully installed Cloudlog on my subdomain and I am very happy with the software. But I can't get any further with one thing. I connected my FT-710 via CloudlogCAT 1.3.0.0 (API code read and whrite) and OmniRig 1.2. If I change the frequency on the FT-710, it is displayed correctly in the QSO window. So far so good. If I now click on a call sign in Cloudlog's DXCluster, the call sign etc. is adopted, but not the new frequency. This means that the previously set old frequency remains in the QSO window and the transceiver. Are there any special settings that I missed?
Many thanks for your help.

73 HB9HUP de Roger

**Comment by phl0** [comment]
Nothing missed. Its just that CAT in CL is currently one way only. So only transferred from radio to CL.

**Comment by GregTroar** [comment]
Is there any plan actually to have the radio follow the cluster frequency or this is not planned ?
Cause this is a bit a deal breaker for me.
Thanks a lot

**Reply by magicbug** [reply]
Please remember this is a web application interacting with the desktop, however yes its planned but cant give a solid ETA.

**Reply by GregTroar** [reply]
OK great thanks
